roachtest: add flaky test cases to ruby-pg ignoreList

spilchen opened the pull request:
This ignores the result of a couple of flaky tests in the ruby-pg suite.

Closes #150127
Closes #149419

Epic: none
Release note: none

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/150429)
<!-- Reviewable:end -->

rafiss reviewed (changes requested):
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/150429)* status: <span>:shipit:</span> complete! 0 of 0 LGTMs obtained (waiting on @spilchen)

___
*[`pkg/cmd/roachtest/tests/ruby_pg_blocklist.go` line 183 at r1](https://reviewable.io/reviews/cockroachdb/cockroach/150429#-OVOXJG0Dgg_nkYp1Gki:-OVOXJG0Dgg_nkYp1Gkj:b3d5mca) ([raw file](https://github.com/cockroachdb/cockroach/blob/040ed67493379cd7758bae5d067892d8f9a8d0a9/pkg/cmd/roachtest/tests/ruby_pg_blocklist.go#L183)):*
> ```Go
> }
> 
> var rubyPGIgnorelist = blocklist{
> ```

let's add the test here, to the ignorelist. this one is for flaky tests that we want to ignore. the `blocklist` is for tests that are expected to fail and which we may resolve by implementing missing functionality.

i also noticed in the linked issue that the test `with a Fiber scheduler connects several times concurrently` failed too, and has failed before in a few other reported issues (https://github.com/cockroachdb/cockroach/issues/149419). i think we should mark that one as ignored also


<!-- Sent from Reviewable.io -->

spilchen committed:
roachtest: add flaky test cases to ruby-pg ignoreList

This ignores the result of a couple of flaky tests in the ruby-pg suite.

Closes #150127
Closes #149419

Epic: none
Release note: none

spilchen reviewed (commented):
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/150429)* status: <span>:shipit:</span> complete! 0 of 0 LGTMs obtained (waiting on @rafiss and @spilchen)

___
*[`pkg/cmd/roachtest/tests/ruby_pg_blocklist.go` line 183 at r1](https://reviewable.io/reviews/cockroachdb/cockroach/150429#-OVOXJG0Dgg_nkYp1Gki:-OVOpWtk7jAgvd_kfFKo:b-atzorz) ([raw file](https://github.com/cockroachdb/cockroach/blob/040ed67493379cd7758bae5d067892d8f9a8d0a9/pkg/cmd/roachtest/tests/ruby_pg_blocklist.go#L183)):*
<details><summary><i>Previously, rafiss (Rafi Shamim) wrote…</i></summary><blockquote>

let's add the test here, to the ignorelist. this one is for flaky tests that we want to ignore. the `blocklist` is for tests that are expected to fail and which we may resolve by implementing missing functionality.

i also noticed in the linked issue that the test `with a Fiber scheduler connects several times concurrently` failed too, and has failed before in a few other reported issues (https://github.com/cockroachdb/cockroach/issues/149419). i think we should mark that one as ignored also
</blockquote></details>

ah good catch. I have gotten these mixed up before.

I also added the additional test as flaky too.


<!-- Sent from Reviewable.io -->
